js/views/userListVw.js

```diff
@@ -26,11 +26,10 @@ module.exports = Backbone.View.extend({
 
   render: function(){
     var self = this;
-    this.listWrapper = $('<div class="flexRow flexExpand border0 custCol-border"></div>');
+    this.listWrapper = $('<div class="list flexRow flexExpand border0 custCol-border"></div>');
     if(this.usersShort.models.length > 0)
     {
-      __.each(this.usersShort.models, function (user)
-      {
+      this.usersShort.each(function (user) {
```

Comment:
Why use the jQuery each instead of the underscore each? Either is fine, I'm just curious what your reason was.